packages/patternfly-4/react-core/src/components/Nav/Nav.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/Nav/nav.css';
+import { css } from '@patternfly/react-styles';
+
+import { Omit } from '../../helpers/typeUtils';
+
+export interface NavProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>, 'onSelect'> {
+  /** Anything that can be rendered inside of the nav */
+  children?: React.ReactNode;
+  /** Additional classes added to the container */
+  className?: string;
+  /** Callback for updating when item selection changes */
+  onSelect?(selectedItem: {groupId: number | string; itemId: number | string; to:string, event: React.FormEvent<HTMLInputElement>}): void;
```

Comment:
Sorry to nitpick, but please change to `onSelect?: (selectedItem: {groupId: number | string; itemId: number | string; to:string, event: React.FormEvent<HTMLInputElement>}) => void;` (use arrow functions for the types).